frontend/src/pages/learn/main-sources-of-retirement-income.tsx

```diff
@@ -334,7 +334,7 @@ const MainSourcesOfRetirementIncome: FC = () => {
           ))}
         </ul>
 
-        <h4 className="h6 pb-3">{t('canada-pension-plan-program.cpp-post-retirement-benefit.header')}</h4>
+        <h4 className="h3 py-3">{t('canada-pension-plan-program.cpp-post-retirement-benefit.header')}</h4>
```

Comment:
Why this `h4` styled as `h6` changed to `h3` style? Other `h4` are still styled as `h6`.

Reply:
This was meant to be switched to an h3, fixed and updated the h4's to include h4 tag instead.